buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -123,7 +125,7 @@
     }
 
     public String getName() {
-        return name;
+        return nameCustomization.apply(name);
```

Comment:
Why not do this in the constructor? Same goes for `safeName()` I suppose too.

Reply:
nameCustomization is mutable, we need to reflect what it's set to in the build script.